apps/clr-ui-tokens/src/app/app.module.ts

```diff
@@ -0,0 +1,87 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+
+import { angleIcon, cogIcon, ClarityIcons, sunIcon, worldIcon, atomIcon } from '@cds/core/icon';
+import { CdsIconModule } from '@cds/angular/icon';
+import { CdsAccordionModule } from '@cds/angular/accordion';
+import { CdsAlertModule } from '@cds/angular/alert';
+import { CdsButtonModule } from '@cds/angular/button';
+import { CdsCheckboxModule } from '@cds/angular/checkbox';
+import { CdsDatalistModule } from '@cds/angular/datalist';
+import { CdsFormsModule } from '@cds/angular/forms';
+import { CdsFileModule } from '@cds/angular/file';
+import { CdsInputModule } from '@cds/angular/input';
+import { CdsPasswordModule } from '@cds/angular/password';
+import { CdsRadioModule } from '@cds/angular/radio';
+import { CdsRangeModule } from '@cds/angular/range';
+import { CdsSearchModule } from '@cds/angular/search';
+import { CdsSelectModule } from '@cds/angular/select';
+import { CdsTagModule } from '@cds/angular/tag';
+import { CdsTextareaModule } from '@cds/angular/textarea';
+import { CdsTimeModule } from '@cds/angular/time';
+import { CdsToggleModule } from '@cds/angular/toggle';
+
+import { ClarityModule } from '@clr/angular';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { DatagridDemoComponent } from './datagrid-demo/datagrid-demo.component';
+import { WizardDemoComponent } from './wizard-demo/wizard-demo.component';
+import { AccordionDemoComponent } from './accordion-demo/accordion-demo.component';
+import { StepperDemoComponent } from './stepper-demo/stepper-demo.component';
+import { ComboboxDemoComponent } from './combobox-demo/combobox-demo.component';
+import { ModalDemoComponent } from './modal-demo/modal-demo.component';
+import { OverviewComponent } from './overview/overview.component';
+
+@NgModule({
```

Comment:
Since https://github.com/vmware-clarity/starters is now set up maybe this should be added there rather than next? Maybe this project can be named something like `angular-clr` in the starters repo since the others are plain Angular + Core. Then we could use this as a minimal example of core + clr-angular.

We may want to remove the apps directory eventually so we don't have duplicate code maintenance/confusion down the road. Thoughts @hippee-lee ?

Reply:
Adding a core + clr-angular is ok with me. I can port this over after the PR merges.

Reply:
That's a fair point, we can leave as is and not port it over to starters. The datagrid will have similar demo apps that are not quite starters as well. Maybe long term those all stay in the core repo under apps.

Reply:
We can have a separate discussion how we do that eventually, its fine for now though.